src/tun/tun_windows.go

```diff
@@ -31,14 +33,24 @@ func (tun *TunAdapter) setup(ifname string, addr string, mtu uint64) error {
 		if guid, err = windows.GUIDFromString("{8f59971a-7872-4aa6-b2eb-061fc4e9d0a7}"); err != nil {
 			return err
 		}
-		if iface, err = wgtun.CreateTUNWithRequestedGUID(ifname, &guid, int(mtu)); err != nil {
-			return err
-		}
-		if !waitForTUNUp(iface.Events()) {
-			return fmt.Errorf("TUN did not come up in time")
+		tun.log.Printf("Creating TUN")
```

Comment:
Maybe don't need so many logging lines.

Reply:
I've removed this line, but I think it is better to leave the others.